rules/windows/powershell/powershell_script/posh_ps_set_acl.yml

```
@@ -0,0 +1,32 @@
+title: PowerShell Set-Acl Windows Folder
+id: cae80281-ef23-44c5-873b-fd48d2666f49
+status: experimental
+description: Detects PowerShell scripts that set ACLs to a file in the Windows folder
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/74438b0237d141ee9c99747976447dc884cb1a39/atomics/T1505.005/T1505.005.md
+author: frack113
+date: 2023/07/18
+tags:
+    - attack.defense_evasion
+    - attack.t1222
+logsource:
+    product: windows
+    category: ps_script
+    definition: bade5735-5ab0-4aa7-a642-a11be0e40872
+detection:
+    selection_cmdlet:
+        ScriptBlockText|contains|all:
+            - 'Set-Acl '
+            - '-Path '
+    selection_path:
+        ScriptBlockText|contains:
+            - ':\Windows\'
+            - ':\Windows/'
+            - ':/Windows\'
+            - ':/Windows/'
+            - '$env:windir\'
+            - '$env:windir/'
```

```suggestion
            - '$env:windir'
            - '$env:windir'
```
I think here we don't necessarily need it. (there is no `$env:windirsomethingelse`)